modules/backend/formwidgets/DatePicker.php

```diff
@@ -46,6 +46,12 @@ class DatePicker extends FormWidgetBase
      */
     public $yearRange = null;
 
+    /**
+     * @var int first day of the week
+     * eg: 0 (= Sunday) or 1 (= Monday)
```

Comment:
This is a very minor change, but please change this comment to read `eg: 0 (Sunday), 1 (Monday), 2 (Tuesday), etc.` to make it more clear that this isn't just a boolean flag, it is an integer option that can support any day being the first day of the week.